.npmrc

```diff
@@ -0,0 +1,2 @@
+ignore-workspace-cycles=true
```

Comment:
This is suppressing dependency warnings that only affect dev dependencies due to test code. Everything still works.